demos/python_demos/human_pose_estimation_3d_demo/human_pose_estimation_3d_demo.py

```diff
@@ -63,6 +63,7 @@ def rotate_poses(poses_3d, R, t):
                       type=str, default=None)
     args.add_argument('--fx', type=np.float32, default=-1, help='Optional. Camera focal length.')
     args.add_argument('--no_show', help='Optional. Do not display output.', action='store_true')
+    args.add_argument('--loop', default=False, action='store_true', help='Optional. Run the input data in a loop.')
```

Comment:
```suggestion
    args.add_argument('-loop', '--loop', default=False, action='store_true', help='Optional. Run the input data in a loop.')
```
Put this argument after `-i`.